lib/jekyll/commands/build.rb

```diff
@@ -70,9 +70,14 @@ def build(site, options)
         # options - A Hash of options passed to the command
         #
         # Returns nothing.
-        def watch(_site, options)
-          External.require_with_graceful_fail "jekyll-watch"
-          Jekyll::Watcher.watch(options)
+        def watch(site, options)
+          External.require_with_graceful_fail 'jekyll-watch'
```

Comment:
Hey! One more thing: the travis build failed for this PR because you're using single quotes here. Our styleguide requests double quotes for all string literals. If you run `script/fmt` locally, you should see this as well. 😄